botx/client/stickers_api/delete_sticker_pack.py

```diff
@@ -0,0 +1,46 @@
+from uuid import UUID
+
+from botx.client.authorized_botx_method import AuthorizedBotXMethod
+from botx.models.api_base import UnverifiedPayloadBaseModel, VerifiedPayloadBaseModel
+
+try:
+    from typing import Literal
+except ImportError:
+    from typing_extensions import Literal  # type: ignore  # noqa: WPS440
+
+
+class BotXAPIDeleteStickerPackRequestPayload(UnverifiedPayloadBaseModel):
+    sticker_pack_id: UUID
+
+    @classmethod
+    def from_domain(
+        cls,
+        sticker_pack_id: UUID,
+    ) -> "BotXAPIDeleteStickerPackRequestPayload":
+        return cls(sticker_pack_id=sticker_pack_id)
+
+
+class BotXAPIDeleteStickerPackResponsePayload(VerifiedPayloadBaseModel):
+    status: Literal["ok"]
+
+
+class DeleteStickerPackMethod(AuthorizedBotXMethod):
+    status_handlers = {
+        **AuthorizedBotXMethod.status_handlers,
+    }
```

Comment:
You don't have to define it empty. It's set by default

Reply:
fixed